app/assets/stylesheets/petitions/views/_petition-show.scss

```diff
@@ -8,14 +8,92 @@
   }
 }
 
+
+// Custom animation
+@-webkit-keyframes fadeInLeft {
+  0% {
+    opacity: 0;
+    -webkit-transform: translate3d(-100%, 0, 0);
+    transform: translate3d(-100%, 0, 0);
+  }
+
+  100% {
+    opacity: 1;
+    -webkit-transform: none;
+    transform: none;
+  }
+}
+
+@keyframes fadeInLeft {
+  0% {
+    opacity: 0;
+    -webkit-transform: translate3d(-100%, 0, 0);
+    transform: translate3d(-100%, 0, 0);
+  }
+
+  100% {
+    opacity: 1;
+    -webkit-transform: none;
+    transform: none;
+  }
+}
+
+%animated {
```

Comment:
Coolio. Better to copy and paste only what we need. If you look at the output you'll see lots of `@keyframe` declarations which do get included.